app/about_us/OurCrew.js

```diff
@@ -0,0 +1,25 @@
+const OurCrew = () => {
+    const crewMembers = [
+      { name: "Alex Johnson", role: "Captain", image: "react-1-hw/app/about_us/Boulder_Worldcup_Vienna_28-05-2010_quali-w100_Alex_Johnson.jpg" },
```

Comment:
These are absolute file paths from your local machine, which won't work in a deployed web app. Try to move the images to the public folder to resolve this :)